langchain/vectorstores/redis.py

```diff
@@ -84,6 +84,33 @@ def add_texts(
     def similarity_search(
         self, query: str, k: int = 4, **kwargs: Any
     ) -> List[Document]:
+        docs_and_scores = self.similarity_search_with_score(query, k=k)
+        return [doc for doc, _ in docs_and_scores]
+
+    def similarity_search_limit_score(
+        self, query: str, k: int = 4, score_thread: float = 0.2, **kwargs: Any
+    ) -> List[Document]:
```

Comment:
would be good to add docstring explaining if documents are kept if the score is less than score_threshold (could also be thought of as above, without knowing more)

Reply:
Okay, I will add the description